### DISCUSSION TITLE
I accidentally lose my place -> session history?

### DISCUSSION THREAD

**Discussion by floatiepen** [advplyr/audiobookshelf-app#439] [Ideas]
Several times, I have accidentally lost my place, and there isn't a good way to recover.  If the audiobook is a single 50+ hour track it's very easy to get lost.  Usually becase I accidentally touch a "skip" button, or the progress slider (over 50+ hours, it's more hazard than help).  Once it was because Android Auto just restarted the book from 0, and I had to guess at where I left off.

Is there a way to see the session history (the start/end points of the N most recent sessions) in the app?  If so, I could choose one of those and quickly get back to listening.

This has happened a couple times while driving, and there is no way to recover until I get home and can log in to the web interface and look at my play-session history.  With an easy way to recover, I could at least pull over and get my book back on track. :D

Even with this occasional snag, I love audiobookshelf!  The mobile apps and server are great and getting beter!

**Comment by advplyr** [comment]
We will implement listen history. Feature request to follow is here https://github.com/advplyr/audiobookshelf-app/issues/466
